test/ruby_lsp_rails/hover_test.rb

```diff
@@ -47,14 +47,8 @@ class User < ApplicationRecord
         RUBY
 
         assert_equal(<<~CONTENT.chomp, response.contents.value)
-          ```ruby
```

Comment:
Just verifying this passes in CI, still need to understand why this has changed.

Reply:
Suspected related to https://github.com/Shopify/ruby-lsp/pull/2322